IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/WolfSSLDemo/src/wolfDemo/wolf_client.c

```diff
@@ -0,0 +1,177 @@
+/* wolf_client.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/ssl.h>
+#include <stdio.h>
+#include <string.h>
+#include "r_t4_itcpip.h"
+#include "wolfssl/certs_test.h"
+#include "wolf_demo.h"
+
+ER    t4_tcp_callback(ID cepid, FN fncd , VP p_parblk);
+
+static int my_IORecv(WOLFSSL* ssl, char* buff, int sz, void* ctx)
+{
+    int ret;
+    ID  cepid;
+
+    if(ctx != NULL)cepid = *(ID *)ctx;
+    else return WOLFSSL_CBIO_ERR_GENERAL;
+
+    ret = tcp_rcv_dat(cepid, buff, sz, TMO_FEVR);
+    if(ret > 0)return ret;
+    else         return WOLFSSL_CBIO_ERR_GENERAL;
+}
+
+static int my_IOSend(WOLFSSL* ssl, char* buff, int sz, void* ctx)
+{
+    int ret;
+    ID  cepid;
+
+    if(ctx != NULL)cepid = *(ID *)ctx;
+    else return WOLFSSL_CBIO_ERR_GENERAL;
+
+    ret = tcp_snd_dat(cepid, buff, sz, TMO_FEVR);
+    if(ret == sz)return ret;
+    else         return WOLFSSL_CBIO_ERR_GENERAL;
+}
+
+static int getIPaddr(char *arg)
+{
+    int a1, a2, a3, a4;
+    if(sscanf(arg, "%d.%d.%d.%d", &a1, &a2, &a3, &a4) == 4)
+         return (a1 << 24) | (a2 << 16) | (a3 << 8) | a4;
+    else return 0;
+}
+
+
+WOLFSSL_CTX *wolfSSL_TLS_client_init()
+{
+
+    WOLFSSL_CTX* ctx;
+#ifndef NO_FILESYSTEM
```

Comment:
Fix indent. Back 4 space here.

Reply:
Correction of review remarks
Insert space